packages/ag-charts-website/src/components/changelog/Changelog.tsx

```diff
@@ -365,7 +352,7 @@ export const Changelog = () => {
                         </Alert>
 
                         <ReleaseVersionNotes
-                            releaseNotes={fixVersion === ALL_FIX_VERSIONS ? undefined : releaseNotesMarkdownContent}
+                            releaseNotes={releaseNotesMarkdownContent}
```

Comment:
Actually, this might not be right...checking

Reply:
Ah, it's defined on line `340`: https://github.com/ag-grid/ag-charts/pull/3253/files#diff-09baf5f9fe9b64ab2d62875d7b9c101064d8e4a2113676593f88b751685cffdfR340